File: programmers/Level01/Programmers_12954.java
package programmers;

class Programmers_12954 {
    public static void main(String[] args) {
        System.out.println(solution(2, 5));
        System.out.println(solution(4, 3));
        System.out.println(solution(-4, 2));
    }

    public static long[] solution(long x, int n){
        long[] answer = new long[n];
        for(int i = 0; i < n; i++){
            answer[i] = x * (i + 1);
        }
        return answer;
    }
}

File: programmers/Swift/Level01/Programmers_12954.swift
final class Programmers_12954 {
    func solution(_ x:Int, _ n:Int) -> [Int64] {
        var arr: [Int64] = []
        for i in 1...n {
            arr.append(Int64(x*i))
        }
        return arr
    }

    func solution2(_ x:Int, _ n:Int) -> [Int64] {
        return Array(1...n).map{ Int64($0 * x)}
    }
}